### PR TITLE
Change of collectibles images source url

### PR DESCRIPTION
**Pull request by LendiHop**

### CODE REVIEW

File: src/utils/image.utils.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const formatCollectibleUri = (collectible: TokenInterface) => {
   const { address } = collectible;
   const id = `${collectible.id}`;
 
-  return `https://assets.objkt.com/file/assets-001/${address}/${id.length > 1 ? id[id.length - 2] : 0}/${
+  return `https://assets.objkt.media/file/assets-001/${address}/${id.length > 1 ? id[id.length - 2] : 0}/${
```
Comment (unixvb) [inline src/utils/image.utils.ts:17]: reuse `objktOrigin`

Comment (unixvb) [reply src/utils/image.utils.ts:17]: (move it out of the tests file)

File: src/utils/image.utils.ts [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import { TokenInterface } from '../token/interfaces/token.interface';
 
-const IPFS_GATEWAY = 'ipfs.io';
+const IPFS_GATEWAY = 'cloudflare-ipfs.com';
 const IPFS_PROTOCOL_PREFIX = 'ipfs://';
 
+export const objktOrigin = 'https://assets.objkt.media/file/assets-001/';
```
Comment (unixvb) [inline src/utils/image.utils.ts:6]: uppercase

File: src/utils/image.utils.ts [hunk 3]
```diff
@@ -14,7 +16,5 @@ export const formatCollectibleUri = (collectible: TokenInterface) => {
   const { address } = collectible;
   const id = `${collectible.id}`;
 
-  return `https://assets.objkt.com/file/assets-001/${address}/${id.length > 1 ? id[id.length - 2] : 0}/${
-    id[id.length - 1]
-  }/${id}/thumb400`;
+  return `${objktOrigin + address}/${id.length > 1 ? id[id.length - 2] : 0}/${id[id.length - 1]}/${id}/thumb400`;
```
Comment (unixvb) [inline src/utils/image.utils.ts:19]: style

File: src/utils/image.spec.ts [hunk 4]
```diff
@@ -17,8 +15,8 @@ describe('image utils', () => {
       expect(formatImgUri(mockHttpsUri)).toEqual(mockHttpsUri);
     });
 
-    it('should convert ipfs:// uri to URL to ipfs.io for that file', () => {
-      expect(formatImgUri('ipfs://mockFileHash')).toEqual('https://ipfs.io/ipfs/mockFileHash/');
+    it('should convert ipfs:// uri to URL to cloudflare-ipfs.com for that file', () => {
+      expect(formatImgUri('ipfs://mockFileHash')).toEqual('https://cloudflare-ipfs.com/ipfs/mockFileHash/');
```
Comment (unixvb) [inline src/utils/image.spec.ts:19]: hardcode

**Review by letier3110 — APPROVED** [review]

**Review by unixvb — APPROVED** [review]

**Pull request merged**